2020-06-22 19:09:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabelle "UPS" nicht gefunden
Table "UPS" not found; SQL statement:
Drop Table ups [42102-200]
2020-06-22 19:09:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabelle "UPS" nicht gefunden
Table "UPS" not found; SQL statement:
Drop Table ups [42102-200]
2020-06-22 19:25:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Feld "STUDIENGANGS_ID" nicht gefunden
Column "STUDIENGANGS_ID" not found; SQL statement:
Create Table studiengang(
studiengang_id INT NOT NULL AUTO_INCREMENT,
name VARCHAR(50) NOT NULL,
kuerzel VARCHAR(10) NOT NULL,
studiengangsleiter VARCHAR(50) NOT NULL,
fakultaet_id INT NOT NULL,
PRIMARY KEY(studiengangs_id),
FOREIGN KEY (fakultaet_id) REFERENCES fakultaet(fakultaet_id)
) [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.table.IndexColumn.mapColumns(IndexColumn.java:115)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:123)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:78)
	at org.h2.command.ddl.CommandWithColumns.createConstraints(CommandWithColumns.java:83)
	at org.h2.command.ddl.CreateTable.update(CreateTable.java:133)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:832)
2020-06-22 19:35:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "CREATE TABLE KURS(
KURS_ID INT NOT NULL AUTO_INCREMENT,
JAHRGANG INT NOT NULL,
NUMMER INT NOT NULL,
RAUM VARCHAR(30) NOT NULL,
EMAILVERTEILER VARCHAR(50)
STUDIENRICHTUNG_ID[*] INT NOT NULL,
PRIMARY KEY(KURS_ID),
FOREIGN KEY (STUDIENRICHTUNG_ID) REFERENCES STUDIENRICHTUNG(STUDIENRICHTUNG_ID)
)"; erwartet "FOR, UNSIGNED, INVISIBLE, VISIBLE, NOT, NULL, AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"
Syntax error in SQL statement "CREATE TABLE KURS(
KURS_ID INT NOT NULL AUTO_INCREMENT,
JAHRGANG INT NOT NULL,
NUMMER INT NOT NULL,
RAUM VARCHAR(30) NOT NULL,
EMAILVERTEILER VARCHAR(50)
STUDIENRICHTUNG_ID[*] INT NOT NULL,
PRIMARY KEY(KURS_ID),
FOREIGN KEY (STUDIENRICHTUNG_ID) REFERENCES STUDIENRICHTUNG(STUDIENRICHTUNG_ID)
)"; expected "FOR, UNSIGNED, INVISIBLE, VISIBLE, NOT, NULL, AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"; SQL statement:
Create Table kurs(
kurs_id INT NOT NULL AUTO_INCREMENT,
jahrgang INT NOT NULL,
nummer INT NOT NULL,
raum VARCHAR(30) NOT NULL,
emailverteiler VARCHAR(50)
studienrichtung_id INT NOT NULL,
PRIMARY KEY(kurs_id),
FOREIGN KEY (studienrichtung_id) REFERENCES studienrichtung(studienrichtung_id)
) [42001-200]
2020-06-22 19:52:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Feld "JAVAKENNTISSE" nicht gefunden
Column "JAVAKENNTISSE" not found; SQL statement:
Create Table Student(
matrikel_nr INT NOT NULL AUTO_INCREMENT,
vorname VARCHAR(50) NOT NULL,
nachname VARCHAR(50) NOT NULL,
javakenntnisse INT NOT NULL,
firma_id INT NOT NULL,
kurs_id INT NOT NULL,
PRIMARY KEY(matrikel_nr),
FOREIGN KEY (kurs_id) REFERENCES kurs(kurs_id),
FOREIGN KEY (firma_id ) REFERENCES firma(firma_id ),
CHECK(javakenntisse<=10)
) [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:190)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:195)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:78)
	at org.h2.command.ddl.CommandWithColumns.createConstraints(CommandWithColumns.java:83)
	at org.h2.command.ddl.CreateTable.update(CreateTable.java:133)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:832)
2020-06-22 19:53:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "CREATE TABLE STUDENT(
MATRIKEL_NR INT NOT NULL AUTO_INCREMENT,
VORNAME VARCHAR(50) NOT NULL,
NACHNAME VARCHAR(50) NOT NULL,
JAVAKENNTNISSE INT NOT NULL,
FIRMA_ID INT NOT NULL,
KURS_ID INT NOT NULL,
PRIMARY KEY(MATRIKEL_NR),
FOREIGN KEY (KURS_ID) REFERENCES KURS(KURS_ID),
FOREIGN KEY (FIRMA_ID ) REFERENCES FIRMA(FIRMA_ID ),

)[*]"; erwartet "identifier"
Syntax error in SQL statement "CREATE TABLE STUDENT(
MATRIKEL_NR INT NOT NULL AUTO_INCREMENT,
VORNAME VARCHAR(50) NOT NULL,
NACHNAME VARCHAR(50) NOT NULL,
JAVAKENNTNISSE INT NOT NULL,
FIRMA_ID INT NOT NULL,
KURS_ID INT NOT NULL,
PRIMARY KEY(MATRIKEL_NR),
FOREIGN KEY (KURS_ID) REFERENCES KURS(KURS_ID),
FOREIGN KEY (FIRMA_ID ) REFERENCES FIRMA(FIRMA_ID ),

)[*]"; expected "identifier"; SQL statement:
Create Table Student(
matrikel_nr INT NOT NULL AUTO_INCREMENT,
vorname VARCHAR(50) NOT NULL,
nachname VARCHAR(50) NOT NULL,
javakenntnisse INT NOT NULL,
firma_id INT NOT NULL,
kurs_id INT NOT NULL,
PRIMARY KEY(matrikel_nr),
FOREIGN KEY (kurs_id) REFERENCES kurs(kurs_id),
FOREIGN KEY (firma_id ) REFERENCES firma(firma_id ),

) [42001-200]
2020-06-22 19:54:15 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabelle "FAKULTÄT" nicht gefunden
Table "FAKULTÄT" not found; SQL statement:
SELECT * FROM FAKULTÄT [42102-200]
2020-06-22 19:59:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "ALTER TABLE STUDIENGANG MODIFY[*] COLLUM
STUDIENGANGSLEITER VARCHAR(100)"; erwartet "., ADD, SET, RENAME, DROP, ALTER"
Syntax error in SQL statement "ALTER TABLE STUDIENGANG MODIFY[*] COLLUM
STUDIENGANGSLEITER VARCHAR(100)"; expected "., ADD, SET, RENAME, DROP, ALTER"; SQL statement:
ALTER TABLE STUDIENGANG MODIFY COLLUM
STUDIENGANGSLEITER VARCHAR(100) [42001-200]
2020-06-22 20:00:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "ALTER TABLE STUDIENGANG 
MODIFY[*] COLUMN STUDIENGANGSLEITER VARCHAR(100)"; erwartet "., ADD, SET, RENAME, DROP, ALTER"
Syntax error in SQL statement "ALTER TABLE STUDIENGANG 
MODIFY[*] COLUMN STUDIENGANGSLEITER VARCHAR(100)"; expected "., ADD, SET, RENAME, DROP, ALTER"; SQL statement:
ALTER TABLE STUDIENGANG 
MODIFY COLUMN STUDIENGANGSLEITER VARCHAR(100) [42001-200]
2020-06-22 20:03:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Feld "STUDIENGANGS_ID" nicht gefunden
Column "STUDIENGANGS_ID" not found; SQL statement:
INSERT INTO STUDIENRICHTUNG(NAME,KUERZEL,STUDIENGANGS_ID)
VALUES('TESTstudiengang','tstStudGNG',1) [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.command.Parser.parseColumn(Parser.java:1450)
	at org.h2.command.Parser.parseColumnList(Parser.java:1435)
	at org.h2.command.Parser.parseInsertGivenTable(Parser.java:1804)
	at org.h2.command.Parser.parseInsert(Parser.java:1749)
	at org.h2.command.Parser.parsePrepared(Parser.java:954)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:832)
2020-06-22 20:14:07 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "SELECT * FROM STUDENT
JOIN FIRMA WHERE FIRMA.FIRMA_ID[*]==STUDENT.FIRMA_ID"
Syntax error in SQL statement "SELECT * FROM STUDENT
JOIN FIRMA WHERE FIRMA.FIRMA_ID[*]==STUDENT.FIRMA_ID"; SQL statement:
SELECT * FROM STUDENT
JOIN Firma where firma.firma_id==student.firma_id [42000-200]
2020-06-22 20:14:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "SELECT * FROM STUDENT
JOIN FIRMA ON FIRMA.FIRMA_ID[*]==STUDENT.FIRMA_ID"
Syntax error in SQL statement "SELECT * FROM STUDENT
JOIN FIRMA ON FIRMA.FIRMA_ID[*]==STUDENT.FIRMA_ID"; SQL statement:
SELECT * FROM STUDENT
JOIN Firma ON firma.firma_id==student.firma_id [42000-200]
2020-06-22 20:21:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabelle "STUDENGANG" nicht gefunden
Table "STUDENGANG" not found; SQL statement:
SELECT * FROM STUDENT
JOIN Firma ON firma.firma_id=student.firma_id
JOIN kurs ON KURS.kurs_id =student.kurs_id
JOIN STUDIENRICHTUNG on STUDIENRICHTUNG.STUDIENRICHTUNG_ID= kurs.STUDIENRICHTUNG_id
JOIN STUDENGANG ON STUDENGANG .STUDENGANG_ID=STUDIENRICHTUNG.STUDENGANG_id [42102-200]
2020-06-22 20:21:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabelle "STUDENGANG" nicht gefunden
Table "STUDENGANG" not found; SQL statement:
SELECT * FROM STUDENT
JOIN Firma ON firma.firma_id=student.firma_id
JOIN kurs ON KURS.kurs_id =student.kurs_id
JOIN STUDIENRICHTUNG on STUDIENRICHTUNG.STUDIENRICHTUNG_ID= kurs.STUDIENRICHTUNG_id
JOIN STUDENGANG ON STUDENGANG.STUDENGANG_ID= STUDIENRICHTUNG.STUDENGANG_id [42102-200]
2020-06-22 20:23:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabelle "FAKULTÄT" nicht gefunden
Table "FAKULTÄT" not found; SQL statement:
SELECT * FROM STUDENT
JOIN Firma ON firma.firma_id=student.firma_id
JOIN kurs ON KURS.kurs_id =student.kurs_id
JOIN Studienrichtung ON Studienrichtung .Studienrichtung_ID= kurs.Studienrichtung_ID
JOIN Studiengang ON Studiengang.Studiengang_ID= STUDIENRICHTUNG.Studiengang_ID
JOIN Fakultät ON Fakultät.Fakultät_ID = Studiengang.Fakultät_ID [42102-200]
2020-06-22 20:23:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Feld "FAKULTÄT.FAKULTAET_ID" nicht gefunden
Column "FAKULTÄT.FAKULTAET_ID" not found; SQL statement:
SELECT * FROM STUDENT
JOIN Firma ON firma.firma_id=student.firma_id
JOIN kurs ON KURS.kurs_id =student.kurs_id
JOIN Studienrichtung ON Studienrichtung .Studienrichtung_ID= kurs.Studienrichtung_ID
JOIN Studiengang ON Studiengang.Studiengang_ID= STUDIENRICHTUNG.Studiengang_ID
JOIN Fakultaet ON Fakultät.Fakultaet_ID = Studiengang.Fakultaet_ID [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:190)
	at org.h2.expression.condition.ConditionAndOr.optimize(ConditionAndOr.java:135)
	at org.h2.command.dml.Select.prepare(Select.java:1209)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:832)
2020-06-23 10:24:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "SELECT * *[*]"
Syntax error in SQL statement "SELECT * *[*]"; SQL statement:
SELECT * * [42000-200]
2020-06-23 10:33:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Feld "TestVName2" nicht gefunden
Column "TestVName2" not found; SQL statement:
INSERT INTO Student(Vorname,Nachname,Javakenntnisse,firma_id,kurs_id)
 VALUES("TestVName2","TestNName2",3,1,1) [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Insert.prepare(Insert.java:327)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:832)
2020-06-23 13:05:03 database: flush
org.h2.message.DbException: Allgemeiner Fehler: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:90)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2787)
	at org.h2.mvstore.MVStore.panic(MVStore.java:441)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:404)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3343)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:95)
	at org.h2.engine.Database.getPageStore(Database.java:2739)
	at org.h2.engine.Database.open(Database.java:769)
	at org.h2.engine.Database.openDatabase(Database.java:319)
	at org.h2.engine.Database.<init>(Database.java:313)
	at org.h2.engine.Engine.openSession(Engine.java:69)
	at org.h2.engine.Engine.openSession(Engine.java:201)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:178)
	at org.h2.engine.Engine.createSession(Engine.java:161)
	at org.h2.engine.Engine.createSession(Engine.java:31)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:336)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:169)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:148)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at HandlersControllers.MainDBConnectTest.main(MainDBConnectTest.java:24)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Allgemeiner Fehler: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:883)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:390)
	... 19 more
2020-06-23 13:09:37 database: flush
org.h2.message.DbException: Allgemeiner Fehler: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:90)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2787)
	at org.h2.mvstore.MVStore.panic(MVStore.java:441)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:404)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3343)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:95)
	at org.h2.engine.Database.getPageStore(Database.java:2739)
	at org.h2.engine.Database.open(Database.java:769)
	at org.h2.engine.Database.openDatabase(Database.java:319)
	at org.h2.engine.Database.<init>(Database.java:313)
	at org.h2.engine.Engine.openSession(Engine.java:69)
	at org.h2.engine.Engine.openSession(Engine.java:201)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:178)
	at org.h2.engine.Engine.createSession(Engine.java:161)
	at org.h2.engine.Engine.createSession(Engine.java:31)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:336)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:169)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:148)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at HandlersControllers.MainDBConnectTest.main(MainDBConnectTest.java:24)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Allgemeiner Fehler: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:883)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:390)
	... 19 more
2020-06-23 13:10:24 database: flush
org.h2.message.DbException: Allgemeiner Fehler: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:90)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2787)
	at org.h2.mvstore.MVStore.panic(MVStore.java:441)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:404)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3343)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:95)
	at org.h2.engine.Database.getPageStore(Database.java:2739)
	at org.h2.engine.Database.open(Database.java:769)
	at org.h2.engine.Database.openDatabase(Database.java:319)
	at org.h2.engine.Database.<init>(Database.java:313)
	at org.h2.engine.Engine.openSession(Engine.java:69)
	at org.h2.engine.Engine.openSession(Engine.java:201)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:178)
	at org.h2.engine.Engine.createSession(Engine.java:161)
	at org.h2.engine.Engine.createSession(Engine.java:31)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:336)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:169)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:148)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at HandlersControllers.MainDBConnectTest.main(MainDBConnectTest.java:24)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Allgemeiner Fehler: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:883)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:390)
	... 19 more
2020-06-23 13:12:07 database: flush
org.h2.message.DbException: Allgemeiner Fehler: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:90)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2787)
	at org.h2.mvstore.MVStore.panic(MVStore.java:441)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:404)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3343)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:95)
	at org.h2.engine.Database.getPageStore(Database.java:2739)
	at org.h2.engine.Database.open(Database.java:769)
	at org.h2.engine.Database.openDatabase(Database.java:319)
	at org.h2.engine.Database.<init>(Database.java:313)
	at org.h2.engine.Engine.openSession(Engine.java:69)
	at org.h2.engine.Engine.openSession(Engine.java:201)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:178)
	at org.h2.engine.Engine.createSession(Engine.java:161)
	at org.h2.engine.Engine.createSession(Engine.java:31)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:336)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:169)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:148)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at test.MainDBConnectTest.main(MainDBConnectTest.java:24)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Allgemeiner Fehler: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:883)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:390)
	... 19 more
2020-06-23 13:12:48 database: flush
org.h2.message.DbException: Allgemeiner Fehler: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:90)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2787)
	at org.h2.mvstore.MVStore.panic(MVStore.java:441)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:404)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3343)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:95)
	at org.h2.engine.Database.getPageStore(Database.java:2739)
	at org.h2.engine.Database.open(Database.java:769)
	at org.h2.engine.Database.openDatabase(Database.java:319)
	at org.h2.engine.Database.<init>(Database.java:313)
	at org.h2.engine.Engine.openSession(Engine.java:69)
	at org.h2.engine.Engine.openSession(Engine.java:201)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:178)
	at org.h2.engine.Engine.createSession(Engine.java:161)
	at org.h2.engine.Engine.createSession(Engine.java:31)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:336)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:169)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:148)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at test.MainDBConnectTest.main(MainDBConnectTest.java:24)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Allgemeiner Fehler: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:883)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:390)
	... 19 more
2020-06-24 15:10:30 database: flush
org.h2.message.DbException: Allgemeiner Fehler: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:90)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2787)
	at org.h2.mvstore.MVStore.panic(MVStore.java:441)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:404)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3343)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:95)
	at org.h2.engine.Database.getPageStore(Database.java:2739)
	at org.h2.engine.Database.open(Database.java:769)
	at org.h2.engine.Database.openDatabase(Database.java:319)
	at org.h2.engine.Database.<init>(Database.java:313)
	at org.h2.engine.Engine.openSession(Engine.java:69)
	at org.h2.engine.Engine.openSession(Engine.java:201)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:178)
	at org.h2.engine.Engine.createSession(Engine.java:161)
	at org.h2.engine.Engine.createSession(Engine.java:31)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:336)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:169)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:148)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at test.MainDBConnectTest.main(MainDBConnectTest.java:24)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Allgemeiner Fehler: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:883)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:390)
	... 19 more
2020-06-24 15:10:48 database: flush
org.h2.message.DbException: Allgemeiner Fehler: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:90)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2787)
	at org.h2.mvstore.MVStore.panic(MVStore.java:441)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:404)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3343)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:95)
	at org.h2.engine.Database.getPageStore(Database.java:2739)
	at org.h2.engine.Database.open(Database.java:769)
	at org.h2.engine.Database.openDatabase(Database.java:319)
	at org.h2.engine.Database.<init>(Database.java:313)
	at org.h2.engine.Engine.openSession(Engine.java:69)
	at org.h2.engine.Engine.openSession(Engine.java:201)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:178)
	at org.h2.engine.Engine.createSession(Engine.java:161)
	at org.h2.engine.Engine.createSession(Engine.java:31)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:336)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:169)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:148)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at test.MainDBConnectTest.main(MainDBConnectTest.java:24)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Allgemeiner Fehler: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:883)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:390)
	... 19 more
2020-06-24 15:20:29 database: flush
org.h2.message.DbException: Allgemeiner Fehler: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:90)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2787)
	at org.h2.mvstore.MVStore.panic(MVStore.java:441)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:404)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3343)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:95)
	at org.h2.engine.Database.getPageStore(Database.java:2739)
	at org.h2.engine.Database.open(Database.java:769)
	at org.h2.engine.Database.openDatabase(Database.java:319)
	at org.h2.engine.Database.<init>(Database.java:313)
	at org.h2.engine.Engine.openSession(Engine.java:69)
	at org.h2.engine.Engine.openSession(Engine.java:201)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:178)
	at org.h2.engine.Engine.createSession(Engine.java:161)
	at org.h2.engine.Engine.createSession(Engine.java:31)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:336)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:169)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:148)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at test.MainDBConnectTest.main(MainDBConnectTest.java:24)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Allgemeiner Fehler: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:883)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:390)
	... 19 more
2020-06-24 15:26:42 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "CREATE USER  USER1 PASWORD[*] USER1"; erwartet "COMMENT, PASSWORD, SALT, IDENTIFIED"
Syntax error in SQL statement "CREATE USER  USER1 PASWORD[*] USER1"; expected "COMMENT, PASSWORD, SALT, IDENTIFIED"; SQL statement:
CREATE USER  user1 PASWORD user1 [42001-200]
2020-06-24 15:26:54 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "CREATE USER  'user1'[*] PASWORD 'user1'"; erwartet "identifier"
Syntax error in SQL statement "CREATE USER  'user1'[*] PASWORD 'user1'"; expected "identifier"; SQL statement:
CREATE USER  'user1' PASWORD 'user1' [42001-200]
2020-06-24 15:27:39 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "CREATE USER GUEST PASWORD[*] 'user1'"; erwartet "COMMENT, PASSWORD, SALT, IDENTIFIED"
Syntax error in SQL statement "CREATE USER GUEST PASWORD[*] 'user1'"; expected "COMMENT, PASSWORD, SALT, IDENTIFIED"; SQL statement:
CREATE USER GUEST PASWORD 'user1' [42001-200]
2020-06-24 15:28:26 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "CREATE USER 'tst'[*]"; erwartet "identifier"
Syntax error in SQL statement "CREATE USER 'tst'[*]"; expected "identifier"; SQL statement:
CREATE USER 'tst' [42001-200]
2020-06-24 15:30:12 database: flush
org.h2.message.DbException: Allgemeiner Fehler: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:90)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2787)
	at org.h2.mvstore.MVStore.panic(MVStore.java:441)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:404)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3343)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:95)
	at org.h2.engine.Database.getPageStore(Database.java:2739)
	at org.h2.engine.Database.open(Database.java:769)
	at org.h2.engine.Database.openDatabase(Database.java:319)
	at org.h2.engine.Database.<init>(Database.java:313)
	at org.h2.engine.Engine.openSession(Engine.java:69)
	at org.h2.engine.Engine.openSession(Engine.java:201)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:178)
	at org.h2.engine.Engine.createSession(Engine.java:161)
	at org.h2.engine.Engine.createSession(Engine.java:31)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:336)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:169)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:148)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at test.MainDBConnectTest.main(MainDBConnectTest.java:24)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Allgemeiner Fehler: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:883)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:390)
	... 19 more
2020-06-24 15:31:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Methode nicht zulässig für eine Abfrage. Erlaubt sind execute oder executeQuery, nicht jedoch executeUpdate
Method is not allowed for a query. Use execute or executeQuery instead of executeUpdate; SQL statement:
SELECT * FROM KURS; [90001-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.command.Prepared.update(Prepared.java:217)
	at org.h2.command.CommandContainer.update(CommandContainer.java:133)
	at org.h2.command.Command.executeUpdate(Command.java:267)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:169)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:126)
	at test.MainDBConnectTest.main(MainDBConnectTest.java:29)
2020-06-24 21:48:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Das Objekt wurde bereits geschlossen
The object is already closed [90007-199]
2020-06-24 21:50:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Das Objekt wurde bereits geschlossen
The object is already closed [90007-199]
2020-06-24 21:50:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Das Objekt wurde bereits geschlossen
The object is already closed [90007-199]
2020-06-24 21:51:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Das Objekt wurde bereits geschlossen
The object is already closed [90007-199]
2020-06-24 21:51:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Das Objekt wurde bereits geschlossen
The object is already closed [90007-199]
2020-06-24 21:53:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Das Objekt wurde bereits geschlossen
The object is already closed [90007-199]
2020-06-24 21:54:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Das Objekt wurde bereits geschlossen
The object is already closed [90007-199]
2020-06-24 22:15:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Feld "ID" nicht gefunden
Column "ID" not found [42122-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:451)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3214)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3287)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:350)
	at test.DatabaseDatamanager.getAll(DatabaseDatamanager.java:42)
	at test.MainDBConnectTest.main(MainDBConnectTest.java:14)
2020-06-25 00:09:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "SELECT * FAKULTAET[*]"
Syntax error in SQL statement "SELECT * FAKULTAET[*]"; SQL statement:
SELECT * FAKULTAET [42000-200]
2020-06-25 00:10:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "INSER[*] INTO FAKULTAET(NAME)
VALUES('Hallo')"; erwartet "INSERT"
Syntax error in SQL statement "INSER[*] INTO FAKULTAET(NAME)
VALUES('Hallo')"; expected "INSERT"; SQL statement:
INSER INTO FAKULTAET(NAME)
VALUES('Hallo') [42001-200]
2020-06-25 00:16:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "SELECT [*]' FIMRA"
Syntax error in SQL statement "SELECT [*]' FIMRA"; SQL statement:
SELECT ' FIMRA [42000-200]
2020-06-25 00:18:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "SELECT * FIRMA[*]"
Syntax error in SQL statement "SELECT * FIRMA[*]"; SQL statement:
SELECT * FIRMA [42000-200]
2020-06-25 00:28:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "SELECT *FROM FAKULTAET
SELEFT *[*] FROM STUDENT"
Syntax error in SQL statement "SELECT *FROM FAKULTAET
SELEFT *[*] FROM STUDENT"; SQL statement:
SELECT *from fakultaet
SELEFT * from Student [42000-200]
2020-06-25 00:28:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "SELEFT[*] * FROM STUDENT"; erwartet "SET, SAVEPOINT, SCRIPT, SHUTDOWN, SHOW"
Syntax error in SQL statement "SELEFT[*] * FROM STUDENT"; expected "SET, SAVEPOINT, SCRIPT, SHUTDOWN, SHOW"; SQL statement:
SELEFT * from Student [42001-200]
2020-06-25 00:28:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "SELECT *FROM FAKULTAET
SELECT[*] * FROM STUDENT"
Syntax error in SQL statement "SELECT *FROM FAKULTAET
SELECT[*] * FROM STUDENT"; SQL statement:
SELECT *from fakultaet
SELECT * from Student [42000-200]
2020-06-25 08:24:17 database: flush
org.h2.message.DbException: Allgemeiner Fehler: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:90)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2787)
	at org.h2.mvstore.MVStore.panic(MVStore.java:441)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:404)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3343)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:95)
	at org.h2.engine.Database.getPageStore(Database.java:2739)
	at org.h2.engine.Database.open(Database.java:769)
	at org.h2.engine.Database.openDatabase(Database.java:319)
	at org.h2.engine.Database.<init>(Database.java:313)
	at org.h2.engine.Engine.openSession(Engine.java:69)
	at org.h2.engine.Engine.openSession(Engine.java:201)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:178)
	at org.h2.engine.Engine.createSession(Engine.java:161)
	at org.h2.engine.Engine.createSession(Engine.java:31)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:336)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:169)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:148)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at test.DatabaseDatamanager.getAll(DatabaseDatamanager.java:34)
	at test.MainDBConnectTest.main(MainDBConnectTest.java:14)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Allgemeiner Fehler: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	... 26 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/falko/OneDrive/Documents/GitHub/WMEJP/wmejpTest.mv.db [1.4.199/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:883)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:390)
	... 20 more
2020-06-25 09:03:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Feld "name" nicht gefunden
Column "name" not found [42122-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:451)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3188)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3287)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:314)
	at test.DatabaseDatamanager.getAll(DatabaseDatamanager.java:70)
	at test.MainDBConnectTest.main(MainDBConnectTest.java:15)
2020-06-25 09:31:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Feld "ID" nicht gefunden
Column "ID" not found; SQL statement:
DELETE FROM KURS WHERE id=1 [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:190)
	at org.h2.command.dml.Delete.prepare(Delete.java:184)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:832)
2020-06-25 13:21:37 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Feld "KURS.NAME" nicht gefunden
Column "KURS.NAME" not found; SQL statement:
SELECT KURS.NAME, STUDIENRICHTUNG.NAME FROM KURS JOIN STUDIENRICHTUNG ON KURS.STUDIENRICHTUNG_ID = STUDIENRICHTUNG.STUDIENRICHTUNG_ID [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Select.prepare(Select.java:1206)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:832)
